torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15214,8 +15214,7 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            variant_test_name='reflect',
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
-           dtypes=all_types_and_complex_and(torch.bfloat16),
-           dtypesIfCUDA=all_types_and_complex_and(torch.half, torch.bfloat16),
+           dtypes=all_types_and_complex_and(torch.bfloat16, torch.half),
```

Comment:
Need to add to `nn.functional.pad` as well.

Reply:
Thanks for the comments. Add it.